lib/Frontend/Frontend.cpp

```diff
@@ -261,233 +262,300 @@ ModuleDecl *CompilerInstance::getMainModule() {
   return MainModule;
 }
 
-void CompilerInstance::performSema() {
-  const FrontendOptions &options = Invocation.getFrontendOptions();
-  const InputFileKind Kind = Invocation.getInputKind();
-  ModuleDecl *MainModule = getMainModule();
-  Context->LoadedModules[MainModule->getName()] = MainModule;
-
-  auto modImpKind = SourceFile::ImplicitModuleImportKind::Stdlib;
+static void
+addAdditionalInitialImportsTo(SourceFile *SF,
+                              ModuleDecl *objCModuleUnderlyingMixedFramework,
+                              ModuleDecl *importedHeaderModule,
+                              SmallVectorImpl<ModuleDecl *> &importModules) {
+  SharedTimer timer("performSema-addAdditionalInitialImportsTo");
```

Comment:
There's no point in timing this; no work is being done.

Reply:
Removed the timer.